applications/rag/metadata.yaml

```diff
@@ -72,36 +60,114 @@ spec:
       - name: dataset_embeddings_table_name
         description: Name of the table that stores vector embeddings for input dataset
         varType: string
+        defaultValue: googlemaps_reviews_db
+      - name: disable_ray_cluster_network_policy
+        description: Disables Kubernetes Network Policy for Ray Clusters for this demo. Defaulting to 'true' aka disabled pending fixes to the kuberay-monitoring module. This should be defaulted to false.
+        varType: bool
+        defaultValue: true
       - name: enable_grafana_on_ray_dashboard
         description: Add option to enable or disable grafana for the ray dashboard. Enabling requires anonymous access.
         varType: bool
         defaultValue: false
+      - name: frontend_add_auth
+        varType: bool
+        defaultValue: false
+      - name: frontend_client_id
```

Comment:
Do we need these if they're not exposed in the UI?

Reply:
they are hidden for now.. they are kept.. so they are in sync with variables..